jenkins/_analysis.py

```diff
@@ -181,6 +191,17 @@ def _get_week_number(timestamp):
     return dt.year * 100 + dt.isocalendar()[1]
 
 
+def _get_datetime(timestamp):
+    """
+    Return the datetime from a jenkins timestamp.
+
+    :param int timestamp: The timestamp to convert in ms since utc
+        to a datetime
+    :return datetime: The corresponding datetime.
+    """
+    return datetime.datetime.fromtimestamp(float(timestamp)/1000)
```

Comment:
:question: _get_week_number could call this function. There's also code
in analyse_data.py that does this calculation.